server/report_builder/src/print.rs

```diff
@@ -243,6 +254,34 @@ pub fn generate_report(
     let config: Config = serde_yaml::from_str(&config_data)
         .map_err(|err| anyhow::Error::msg(format!("Failed to parse config file: {}", err)))?;
 
+    let inner_data = ReportGenerateData {
+        report,
+        config,
+        store_id,
+        store_name,
+        output_filename,
+        format,
+        data_id,
+        arguments,
+    };
```

Comment:
Felt like a bit unnecessary structuring and restructuring; did this to minimise code changes in PR.